### DISCUSSION TITLE
Updating to Lit 3.0

### DISCUSSION THREAD

**Discussion by riovir** [ing-bank/lion#2111] [General]
Hi there, are there any near future plans for using Lit 3? I mostly ask to better time our own Lit upgrade.

**Comment by gerjanvangeest** [comment]
@riovir No concrete plans yet, but we are having some internal discussions about it. We'll keep you update if we have more info about it

**Comment by ErhanKurnaz** [comment]
Any update on this?

**Comment by gerjanvangeest** [comment]
@ErhanKurnaz, Yes we have an update. lit 3 is coming to lion somewhere in January.
We are probably releasing an alpha version any time soon.

**Reply by ErhanKurnaz** [reply]
Sick! Thanks for the reply
